backend/src/apiserver/main.go

```diff
@@ -106,12 +106,23 @@ func main() {
 	}
 	log.SetLevel(level)
 
+	go reconcileSwfCrs(resourceManager)
+
 	go startRpcServer(resourceManager)
 	startHttpProxy(resourceManager)
 
 	clientManager.Close()
 }
 
+func reconcileSwfCrs(resourceManager *resource.ResourceManager) {
+	ctx, cancel := context.WithTimeout(context.Background(), 3*time.Minute)
```

Comment:
I don't think we need a timeout context here since we don't really care if it takes longer than 3 minutes when it's run asynchronously. A regular background context should be okay.

Reply:
Done.